crm_lead_product/README.rst

```diff
@@ -0,0 +1,92 @@
+======================
+Lead Product Reference
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcrm-lightgray.png?logo=github
+    :target: https://github.com/OCA/crm/tree/12.0/crm_lead_product
+    :alt: OCA/crm
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/crm-12-0/crm-12-0-crm_lead_product
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/111/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+
+This module allows the user to link multiple products, product categories or product templates to a lead or an opportunity
+in order to be able to do a product demand forecasting taking into account the leads and opportunities defined in odoo.
+
+The module also includes a reporting section where the user can easily see in a pivot and graph view the relationship
+between products, stage of the lead/opportunity and the planned or expected revenue for that product.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+
+To configure this module to be available for the Leads the user needs to activate the Leads option in CRM settings
+
+Usage
+=====
+
+
+* Go to menu **CRM > Lead** and create or edit a Lead adding all the products associated to it.
+* Go to menu **CRM > My Pipeline** and create or edit an Opportunity adding all the products associated to it.
+
```

Comment:
I'd add here the option of running the report "Pipeline by product"

Reply:
Done :+1: